index.md

```diff
@@ -63,4 +76,132 @@ kubectl apply -f https://raw.githubusercontent.com/openebs/cstor-operators/maste
+| cstorCSIPlugin.image.repository | string | `"openebs/cstor-csi-driver"` |  CStor CSI driver image repository |
+| cstorCSIPlugin.image.tag | string | `"2.5.0"` | CStor CSI driver image tag |
+| cstorCSIPlugin.name | string | `"cstor-csi-plugin"` | CStor CSI driver container name |
+| cvcOperator.annotations | object | `{}` | CVC operator annotations |
+| cvcOperator.componentName | string | `"cvc-operator"` | CVC operator component name |
+| cvcOperator.image.pullPolicy | string | `"IfNotPresent"` | CVC operator image pull policy  |
+| cvcOperator.image.registry | string | `nil` | CVC operator image registry |
+| cvcOperator.image.repository | string | `"openebs/cvc-operator"` | CVC operator image repository |
+| cvcOperator.image.tag | string | `"2.5.0"` | CVC operator image tag |
+| cvcOperator.nodeSelector | object | `{}` | CVC operator pod nodeSelector |
+| cvcOperator.podAnnotations | object | `{}` | CVC operator pod annotations |
+| cvcOperator.resources | object | `{}` |CVC operator pod resources  |
+| cvcOperator.resyncInterval | string | `"30"` | CVC operator resync interval |
+| cvcOperator.securityContext | object | `{}` | CVC operator security context |
+| cvcOperator.target.image.registry | string | `nil` | Volume Target image registry  |
+| cvcOperator.target.image.repository | string | `"openebs/cstor-istgt"` | Volume Target image repository |
+| cvcOperator.target.image.tag | string | `"2.5.0"` | Volume Target image tag |
+| cvcOperator.tolerations | list | `[]` | CVC operator pod tolerations |
+| cvcOperator.volumeExporter.image.registry | string | `nil` | Volume exporter image registry |
+| cvcOperator.volumeExporter.image.repository | string | `"openebs/m-exporter"` | Volume exporter image repository |
+| cvcOperator.volumeExporter.image.tag | string | `"2.5.0"` | Volume exporter image tag |
+| cvcOperator.volumeMgmt.image.registry | string | `nil` | Volume mgmt image registry |
+| cvcOperator.volumeMgmt.image.repository | string | `"openebs/cstor-volume-manager"` | Volume mgmt image repository |
+| cvcOperator.volumeMgmt.image.tag | string | `"2.5.0"` |  Volume mgmt image tag|
+| imagePullSecrets | string | `nil` | Image registry pull secrets |
+| openebsNDM.enabled | bool | `true` | Enable OpenEBS NDM dependency |
+| rbac.create | bool | `true` | Enable RBAC |
+| rbac.pspEnabled | bool | `false` | Enable PodSecurityPolicy |
+| release.version | string | `"2.5.0"` | Openebs CStor release version |
+| serviceAccount.annotations | object | `{}` | Service Account annotations |
+| serviceAccount.csiController.create | bool | `true` | Enable CSI Controller ServiceAccount |
+| serviceAccount.csiController.name | string | `"openebs-cstor-csi-controller-sa"` | CSI Controller ServiceAccount name |
+| serviceAccount.csiNode.create | bool | `true` | Enable CSI Node ServiceAccount |
+| serviceAccount.csiNode.name | string | `"openebs-cstor-csi-node-sa"` | CSI Node ServiceAccount name |
```

Comment:
Here after the table we can show example , how to use the --set command with helm command.

Reply:
Simliar to what we do in openebs/chart like:

---

Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.

Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,

```bash
helm install --name openebs-cstor -f values.yaml openebs-cstor/cstor 
```